### PR TITLE
lib: RoundCorner*Layout: make round corner color xml themable.

### PR DESCRIPTION
**Pull request by tribalfs**
Also added util to set round corner color of preference screen

### CODE REVIEW

File: lib/src/main/java/dev/oneuiproject/oneui/utils/PreferenceUtils.java [hunk 1]
```diff
@@ -65,4 +72,36 @@ public static void addRelatedCardToFooter(@NonNull PreferenceFragmentCompat frag
         }
     }
 
+    public void applyThemeRoundCornerColor(@NonNull PreferenceFragmentCompat preferenceFragment){
```
Comment (Lemkinator) [inline lib/src/main/java/dev/oneuiproject/oneui/utils/PreferenceUtils.java:75]: would prefer "applyThemeRound**ed**CornerColor"

File: lib/src/main/java/dev/oneuiproject/oneui/utils/PreferenceUtils.java [hunk 2]
```diff
@@ -65,4 +72,36 @@ public static void addRelatedCardToFooter(@NonNull PreferenceFragmentCompat frag
...
+
+        TypedValue typedValue = new TypedValue();
+        if (preferenceFragment.getContext().getTheme().resolveAttribute(R.attr.roundedCornerColor, typedValue, true)) {
+            int roundColor = typedValue.data;
+            setPreferenceRoundCornerColor(preferenceFragment, roundColor);
+        }
+    }
+
+    public void  setPreferenceRoundCornerColor(@NonNull PreferenceFragmentCompat preferenceFragment, @ColorInt int color){
```
Comment (Lemkinator) [inline lib/src/main/java/dev/oneuiproject/oneui/utils/PreferenceUtils.java:84]: would prefer "setPreferenceRound**ed**CornerColor"

**Comment by ronaldoguedess** [comment]
!important!!!

**Comment by ronaldoguedess** [comment]
> Also added util to set round corner color of preference screen

Amazing! it was missing!

**Comment by ronaldoguedess** [comment]
> Also added util to set round corner color of preference screen

It would be a valuable addition OneUi lib. As I mentioned before, I'm currently working on transitioning my app's design to a purple theme, and having this feature implemented would align perfectly with my goals. I appreciate your recommendation and I'm definitely interested in the feature.

**Pull request merged**